.github/workflows/ci-conda.yml

```diff
@@ -77,17 +77,17 @@ jobs:
         with:
           path: ~/conda_pkgs_dir
           key:
-            ${{ runner.os }}-conda-${{ hashFiles('src/environment.yml') }}
+            ${{ runner.os }}-conda-${{ hashFiles('src/environment-3.11.yml') }}
 
       - name: Setup Conda
         uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: ${{ matrix.python }}
           mamba-version: "*"
           channels: conda-forge,defaults
-          channel-priority: true
+          channel-priority: strict
```

Comment:
Could the time difference come from this change?

Reply:
I had to use "strict" to eliminate some earlier error.

Reply:
What kind of errors?